assets/mongodb_queries/data_qc.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# nmdc_schema/assets/mongodb_queries/referential_integrity_check.py
+"""
+This module contains the MongoDB queries for checking referential integrity.
+"""
+import click
+import logging
+import os
+from pymongo import MongoClient
+
+@click.group()
+@click.argument("mongo_uri", type=str)
+@click.pass_context
+def cli(ctx, mongo_uri):
+    ctx.ensure_object(dict)
+    ctx.obj["MONGO_URI"] = mongo_uri
+
+@cli.command()
+@click.pass_context
+def referential_integrity(ctx):
+    """
+    Check referential integrity of slots in the MongoDB database using the queries in
+    in nmdb_schema/assets/mongodb_queries/data_qc.
+
+    Check for referential integrity of the following slots:
+        - part_of
+        - has_input
+        - has_output
+        - was_informed_by
+        - metagenome_annotation_id
```

Comment:
i added info in the ticket on how to add checks for was_generated_by and has_calibration. Please add those to this python script.

Reply:
Added queries to check was_generated_by and has_calibration relations.  I am pushing the scripting / automation of these queries to a separate issue